ayekan/exporters/blackbox-exporter/helm-values.yaml

```diff
@@ -14,19 +14,25 @@ serviceMonitor:
   selfMonitor:
     enabled: true
     labels:
-      monitor: enabled
+      lsst.io/monitor: "true"
   defaults:
     labels:
-      monitor: enabled
+      lsst.io/monitor: "true"
   targets:
     - name: googlehttpcheck
       url: https://google.com
       hostname: google.com
       interval: 60s
       module: http_2xx
 
+# TODO: Error: UPGRADE FAILED: release blackbox-exporter failed, and has been
+# rolled back due to atomic being set: failed to create resource: admission
+# webhook "prometheusrulemutate.monitoring.coreos.com" denied the request: Cannot
+# unmarshal rules from spec
```

Comment:
This error shouldn't come up under fleet as we switched from atomic to fleets equivalent of `-wait-for-jobs`.

Reply:
I think the issue is that the `PrometheusRule` objects inside the blackbox-exporter is breaking on the version of the Prometheus operator. The error code is copy/pasted from the manual debugging session, so the `atomic` output might be incorrect. I'll correct the TODO so it correctly reflects the problem (since there rules still break when deploying with rancher).